fastlane_core/lib/fastlane_core/ui/errors/fastlane_error.rb

```diff
@@ -26,12 +26,12 @@ def could_contain_pii?
 
 class Exception
   def fastlane_crash_came_from_custom_action?
-    custom_frame = exception.backtrace.find { |frame| frame.start_with?('actions/') }
```

Comment:
I don't know where it comes from. It happens when you have a syntax error in a dynamically loaded file, e.g. a fastlane plugin. I remember I ran into the same thing when I built danger